### PR TITLE
AST-2270 - Accessibility option

### PR DESCRIPTION
**Pull request by nathanrodrigues2111**
### Description
Jira : https://brainstormforce.atlassian.net/browse/AST-2270

### Screenshots
https://d.pr/i/UpUHWU

### Types of changes
New feature (non-breaking change which adds functionality) 

### How has this been tested?
-Check blog and single page 
- Check in frontend 
- Check if all the options added worked properly

### Checklist:
- [x] My code is tested
- [x] My code passes the PHPCS tests
- [x] My code follows accessibility standards 
- [x] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request 

[AST-2270]: https://brainstormforce.atlassian.net/browse/AST-2270?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: inc/class-astra-dynamic-css.php [hunk 1]
```diff
@@ -731,6 +731,43 @@ public static function return_output( $dynamic_css, $dynamic_css_filtered = '' )
 				);
 			}
```
Comment (GauravKhupse) [inline inc/class-astra-dynamic-css.php:733]: Please add changelog

Comment (nathanrodrigues2111) [reply inc/class-astra-dynamic-css.php:733]: Added @GauravKhupse

**Review by yssambare12 — COMMENTED** [review]
@nathanrodrigues2111 please resolve the PR conflicts

**Review by neerajmasai — APPROVED** [review]

**Review by yssambare12 — APPROVED** [review]

File: inc/customizer/configurations/accessibility/class-astra-accessibility-configs.php [hunk 2]
```diff
@@ -0,0 +1,157 @@
+<?php
+/**
+ * Accessibility Options for Astra Theme.
+ *
+ * @package     Astra
+ * @author      Astra
+ * @copyright   Copyright (c) 2021, Astra
```
Comment (imnavanath) [inline inc/customizer/configurations/accessibility/class-astra-accessibility-configs.php:7]:
```suggestion
 * @copyright   Copyright (c) 2023, Astra
```

**Review by imnavanath — APPROVED** [review]

**Pull request merged**